lib/src/async/metronome.dart

```diff
@@ -98,6 +98,6 @@ class Metronome extends Stream<DateTime> {
   _tickDate() {
     DateTime now = clock.now();
     _controller.add(now);
```

Comment:
Inline the local `now` and add a comment so someone like me doesn't wonder wtf is going on here :)